features/0025-didcomms-transports/README.md

```diff
@@ -0,0 +1,81 @@
+# 0025: Agent Transports
+- Author: Sam Curren <sam@sovrin.org>
+- Start Date: 2019-02-26
+
+## Status
+
+- Status: [PROPOSED](/README.md#rfc-lifecycle)
+- Status Date: 2019-05-27
+- Status Note: Supersedes [INDY PR 94](https://github.com/hyperledger/indy-hipe/pull/94)
+
+## Summary
+
+This RFC Details how different transports are to be used for Agent Messaging.
+
+## Motivation
+
+Agent Messaging is designed to be transport independent, including message encryption and agent message format. Each transport does have unique features, and we need to standardize how the transport features are (or are not) applied. 
+
+## Reference
+
+Standardized transport methods are detailed here. 
+
+### HTTP(S)
+
+HTTP(S) is the first transport for DID Communication that has received heavy attention.
+
+- Messages are transported via HTTP POST.
+- The MIME Type for the POST request is `application/ssi-agent-wire`.
```

Comment:
Is this mime type remaining as "-wire"?

Reply:
I suggest that we rename this to `application/didcomm-enc-envelope` or similar.

Reply:
I reversed envelope-enc, but this is great.